pkg/builder/builder.go

```diff
@@ -669,6 +693,18 @@ func buildPodSpec(vdb *vapi.VerticaDB, sc *vapi.Subcluster) corev1.PodSpec {
 	}
 }
 
+// buildScrutinizePodSpec creates a PodSpec for the scrutinize pod
+func buildScrutinizePodSpec(vscr *v1beta1.VerticaScrutinize) corev1.PodSpec {
+	return corev1.PodSpec{
```

Comment:
We should set the restartPolicy to Never. I think we should try at most once to collect the scrutinize. Maybe have an annotation to allow this to be overridden.